bin/build-drivers/build_drivers.clj

```diff
@@ -0,0 +1,22 @@
+(ns build-drivers
+  "Entrypoint for `bin/build-drivers.sh`. Builds all drivers, if needed."
+  (:require [build-drivers
+             [build-driver :as build-driver]
+             [common :as c]
+             [util :as u]]
+            [clojure.java.io :as io]
+            [colorize.core :as colorize]))
+
+(defn- build-drivers! []
+  (u/step "Building all drivers"
+    (doseq [driver (map keyword (.list (io/file (c/filename c/project-root-directory "modules" "drivers"))))]
+      (build-driver/build-driver! driver))
+    (u/announce "Successfully built all drivers.")))
+
+(defn -main []
+  (try
```

Comment:
The pattern
`(try
     ~@body
     (System/exit 0)
     (catch ...)`
repeats enough that it might warrant a macro

Reply:
yeah. That stuff is shared with some of the stuff in the Mac app build script and the new release script I'm working on as well (https://github.com/metabase/metabase/pull/12860) so it probably makes sense to spin off some of this shared stuff into a lib